app/assets/javascripts/app/error_tracking/error_tracking.js

```diff
@@ -0,0 +1,32 @@
+(function() {
+  'use strict';
+
+  angular
+    .module('council.error_tracking', [])
+    .run(errorTracking);
```

Comment:
I think maybe this should be a service that can be initialised on the main module, and not on inclusion.  @azevedo-252 what dou you think? I'm not sure about if my idea makes sense.